pkgs/by-name/ew/eww/package.nix

```diff
@@ -32,18 +47,34 @@ rustPlatform.buildRustPackage rec {
     librsvg
   ];
 
-  cargoBuildFlags = [ "--bin" "eww" ];
+  cargoBuildFlags = [
+    "--bin"
+    "eww"
+  ];
 
   cargoTestFlags = cargoBuildFlags;
 
   # requires unstable rust features
   RUSTC_BOOTSTRAP = 1;
 
   meta = {
-    description = "ElKowars wacky widgets";
+    description = "A widget system made in Rust to create widgets for any WM";
+    longDescription = ''
+      Eww (ElKowar's Wacky Widgets) is a widget system made in Rust, which lets
```

Comment:
This comma feels wrong to me but I am no English native speaker so I could be mistaken

Reply:
Well, I've always used it that way :)
better to have another opinion

Reply:
I agree with both of w-lfchen's suggestions about the description.

Reply:
Done.